src/app/routes/discussionsForum.js

```diff
@@ -149,10 +149,15 @@ function checkEmail() {
             }
     }
 }
-
+function addHeaders() {
+    return function (proxyReqOpts, srcReq) { 
```

Comment:
Call 
proxyUtils.decorateRequestHeaders(discussions_middleware),
and then ovviride Authorization with below code.